eos_data_distribution/ndn/chunks.py

```diff
@@ -34,9 +34,19 @@ def get_chunk_component(name):
     # The chunk component of a name is the last part...
     return name.get(-1)
 
+def get_chunkless_name(name):
```

Comment:
I haven’t really said it anywhere else yet, but in general it would be nice to see more pydoc comments around,  especially on functions like this which do something fairly impenetrable. They’re easy enough to write quickly when putting the function together. :-)